jetty-http2/http2-server/src/main/java/org/eclipse/jetty/http2/server/HttpChannelOverHTTP2.java

```diff
@@ -261,31 +275,277 @@ public InvocationType getInvocationType()
             {
                 return callback.getInvocationType();
             }
-        });
+        };
+        boolean wakeup = _contentDemander.onContent(content);
```

Comment:
So this is not really `wakeup` at this point.   It's more like `contentWasNeeded` or just `needed'

Reply:
Done.